source/index.html

```diff
@@ -1,12 +1,333 @@
+              питание.
+            </p>
+          </li>
+          <li class="explanation-list__item">
+            <img
+              src="https://via.placeholder.com/60x60"
+              width="60"
+              height="60"
+              alt="Ждать придётся недолго"
+              class="explanation-list__icon"
+            />
+            <p class="text explanation-list__text">
+              Уже через месяц наслаждайтесь изменениями к лучшему вашего
+              питомца!
+            </p>
+          </li>
+        </ul>
+      </section>
+      <section class="product-demonstration">
+        <h2 class="title product-demonstration__title">Живой пример</h2>
+        <p class="text product-demonstration__text">
+          Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на
+          Cat Energy Slim. Отличный результат без изнуряющих тренировок! При
+          этом он не менял своих привычек и по-прежнему спит по 16 часов в день.
+        </p>
+        <ul class="product-demonstration__stats stats-list">
+          <li class="stats-list__item">
+            <span class="stats-list__value">5</span>
+            <small class="title stats-list__subtitle">cнижение веса</small>
+          </li>
+          <li class="stats-list__item">
+            <span class="stats-list__value">60 дней</span>
+            <small class="title stats-list__subtitle">затрачено времени</small>
+          </li>
+        </ul>
+        <p class="product-demonstration__caption">
+          Затраты на питание: 15 000 руб.
+        </p>
+        <div class="product-demonstration__progress progress">
+          <div class="progress__image-slides">
+            <ul class="progress__slides-list">
+              <li class="progress__result progress__result--before">
+                <img
+                  src="https://via.placeholder.com/311x232"
+                  width="311"
+                  height="232"
+                  alt="Толстый котик,который долгое время сидел на обычном корме"
+                  class="progress__image"
+                />
+              </li>
+              <li class="progress__result progress__result--after">
+                <img
+                  src="https://via.placeholder.com/311x232"
+                  width="311"
+                  height="232"
+                  alt="Тот же котик,но уже похудевший,прыткий,как будто заново родился!"
+                  class="progress_image"
+                />
+              </li>
+            </ul>
+          </div>
+          <div class="progress__slider-controls">
+            <button
+              class="progress__slider-button"
+              aria-label="Кнопка переключения слайдера"
+            >
+              Было
+            </button>
+            <input type="range" class="progress__input" />
+            <div class="progress__slider-line"></div>
+            <div class="progress__slider-thumb"></div>
```

Comment:
```suggestion
 <div class="progress__slider-line">
   <div class="progress__slider-thumb"></div>
</div>
            
```

line это белый контейнер, а thumb это зеленый бегунок-индкатор внутри